utils/fdb_wrapper_cpp/src/fdbcs.cpp

```diff
@@ -19,10 +19,20 @@
 #include <iostream>
 #include <thread>
 #include <chrono>
-#include "../include/fdbcs.hpp"
+#include <boost/uuid/uuid.hpp>
+#include <boost/uuid/uuid_io.hpp>
+#include <boost/uuid/random_generator.hpp>
+#include <boost/lexical_cast.hpp>
+#include <chrono>
+#include "fdbcs.hpp"
 
 namespace FDBCS
 {
+
+#define RETURN_ON_ERROR(retcode) \
+  if (!retcode)                  \
+    return retcode;
+
```

Comment:
I like this variant because it is less error-prone:
```suggestion
#define RETURN_ON_ERROR(retcode) do {\
    typeof(retcode) rc = (retcode);  \
    if (!rc) {                       \
      return (rc);                   \
    }                                \
  } while (false)
```

For example, with this macro, we will not do the `commitKeys` twice if an error occurs on line 350.

Reply:
Thanks, fixed